### DISCUSSION TITLE
Gopro 11 shuts down while recording (sometimes)

### DISCUSSION THREAD

**Discussion by carbendi** [gopro/labs#785] [Q&A] [answered]
Hi
While recording my gopro 11 shuts down sometimes. I thought it is about the sd card but different sd cards had the same problem. 
Would be great to understand the reason behind this. (maybe I did something wrong)
It is not happening everytime so and I don't know why which makes it a bit unrealible at the moment. 

[qrlog.txt](https://github.com/gopro/labs/files/14413414/qrlog.txt)
[version.txt](https://github.com/gopro/labs/files/14413415/version.txt)
[mdb_11.log](https://github.com/gopro/labs/files/14413417/mdb_11.log)
[qr.bin.txt](https://github.com/gopro/labs/files/14413420/qr.bin.txt)

**Comment by dnewman-gpsw** [comment, marked as answer]
You are doing a few odd things.

Use !RESET to clear all Labs settings.  You probably should do a factory reset as well.  

Also update your firmware to v2.20.70 (for HERO11.)

If your commands are long, you can break things, and your commands are so long they almost don't fit in the logging messages.

I'm seeing command like this: 
```!MLOGB=200,-128mV3r5Xp25fWe0hS0dR0wNx0sLcFb1d1i2M2S180g0dVoV0oB9oW0oD4oCoSv0q0oR1```

You only ever need to set some settings once, you are setting them hundreds of times.   These are global camera settings: g0dVoV0oB9oW0oD4oCoSv0q0oR1   Set once.

The video mode settings are also set once, as they are saved into the preset
mV3r5Xp25fWe0hS0dR0wNx0sLcFb1d1i2M2S180  (all set once)

Why set ISO Min and Max to 200, with a 180 degree shutter.  Is this for FPV after sunset, or with a lot of ND?

Don't combine !Mwxy= commands with settings.  There is rarely any reason to do that.

After you reset you camera, set you Labs extension is one command like this

!MBITR=180!MWAKE=2!MNR01=1!MGCMP=1!MWIDE=1

You are using EXPT in way not compatible with a fixed shutter, so careful with some combinations.
You setting LOGB=200,-128, that is bad idea, that super flat "look" has been removed as an example.  This is no gain using any offset, lifted black has no real purpose.  LOGB=400 is the recommendation for most use cases (and for existing LUTs.)

This seems it might be a naked camera, or a least without original battery, as the clock keeps resetting.  All these settings look very like for drone applications.  The camera is crashing a bit, so hopeful the resets will fix that.

**Reply by carbendi** [reply]
Thank you for the detailed explenation.
I will do both labs and factory resets, firmware update and will follow your advice on the codes.  

_"Why set ISO Min and Max to 200, with a 180 degree shutter. Is this for FPV after sunset, or with a lot of ND?"_
It was set like this for a commercial shoot to be able to match the colors and exposure to a Alexa camera. 

_"You setting LOGB=200,-128, that is bad idea, that super flat "look" has been removed as an example. This is no gain using any offset, lifted black has no real purpose. LOGB=400 is the recommendation for most use cases (and for existing LUTs.)"_
I experimented a bit and it was easier for me to match gopro footage to Red Helium Camera's RedWidegamutRGB / L3G10. But I will try using LOGB=400 again to see if I can make that work. 
here is the result https://www.instagram.com/p/C20I-LnopmS/?utm_source=ig_web_copy_link&igsh=MzRlODBiNWFlZA==

_"This seems it might be a naked camera, or a least without original battery, as the clock keeps resetting. All these settings look very like for drone applications. The camera is crashing a bit, so hopeful the resets will fix that."_
Yes it is a naked gopro11 on a small fpv drone. Hopefully the changes you mention will solve the problem. 

Thank you again :)

**Comment by ellielo** [comment]
Since you’ve already tested with different SD cards and the issue still occurs, the cause might be something else — like overheating, firmware glitches, battery issues, or even resolution/frame rate settings that demand more processing power. This article does a great job breaking down the possible reasons and solutions: [https://www.linkedin.com/pulse/gopro-shuts-off-while-recording-macgasm-oxbie](https://www.linkedin.com/pulse/gopro-shuts-off-while-recording-macgasm-oxbie). It might help pinpoint exactly what’s going on with your device. Hopefully, it makes your GoPro more reliable moving forward
